### PR TITLE
setup/teardown -> before/after

### PR DESCRIPTION
**Pull request by grosser**
@zendesk/samson 

saw a few weird behaviors when mixing before and setup ... so let's unify on 1 way of doing things ...

### Risks
- None

### CODE REVIEW

File: test/models/buddy_check_test.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@
     before do
       stage.update_attribute(:production, true)
       job_execution.stubs(:execute!)
-      job_execution.stubs(:setup!).returns(true)
+      job_execution.stubs(:before!).returns(true)
```
Comment (pschambacher) [inline test/models/buddy_check_test.rb]: not so sure about this one?

**Comment by pschambacher** [comment]
:+1: except for derp find/replace

File: test/models/buddy_check_test.rb (hunk 1 above)
Comment (grosser) [inline test/models/buddy_check_test.rb]: good catch! ... apparently the test was green even with that ...

Comment (grosser) [inline test/models/buddy_check_test.rb]: fixed!

**Comment by pschambacher** [comment]
:+1:

**Pull request merged**